bin/release.sh

```diff
@@ -0,0 +1,2 @@
+rake db:migrate
```

Comment:
you need to do `chmod +x ./bin/release.sh` git add/commit/push, so that this file is executable

Reply:
thanks!